Topology.dic

```diff
@@ -2038,4 +1967,18 @@ save_
 
        Changed the _type.source attribute of all SU data items to 'Related'.
        (A. Vaitkus)
+;
+         0.9.7                    2024-02-20
```

Comment:
My reservation here is that version 0.9.6 of 2021-11-16 has been at least exposed on the IUCr website at https://www.iucr.org/resources/cif/dictionaries/cif_topology. Since this revision removes two data items, I thought it wise to bump the advertised version number. I know this is in conflict with current COMCIFS workflow, but I hope that we can properly bring IUCr practice into line with the GitHub procedures as we approach Volume G publication.

Reply:
Ok, makes perfect sense. But then the last changelog comment from version 0.9.6 should be moved to 0.9.7 since those changes were not present in version 0.9.6 published on the IUCr website. I will make a change suggestion for this.

Reply:
This conversation can be marked as resolved. Normally,I would do that myself, but it seems that I cannot do it on this repo since I lack write permissions. @nautolycus as the issuer of this PR you should be able to do it.